jenkins-pipelines/continueous-delivery-jenkinsfile

```diff
@@ -0,0 +1,62 @@
+pipeline {
+    agent any
+    environment {
+
+        DEV_USER = 'xxxxx'
+        DEV_HOST = 'xxxxxxx.com'
+        PRE_PROD_USER = 'xxxxx'
+        PRE_PROD_HOST = 'xxxxxx.com'
+        PROD_USER = 'xxxxx'
+        PROD_HOST = 'xxxxxxx.com'
```

Comment:
We should inject these values via envInject plugin. May be setting envInject can over-ride the environment variables set in this file.